### PR TITLE
Rename toc node: Customer relationship management > Partner customer administration

### PR DESCRIPTION
**Pull request by JulCsc**

### CODE REVIEW

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
Thanks for your fixes, @JulCsc . I added one comment, please take a look.

File: api-reference/beta/toc.yml [hunk 1]
```diff
@@ -11863,7 +11863,7 @@ items:
       - name: Overview
         displayName: Azure AD, Microsoft identity platform, Identity and access
         href: resources/azure-ad-overview.md
-      - name: Customer relationship management
```
[the remark this answers is not in the document]
Comment (JulCsc) [reply api-reference/beta/toc.yml:11866]: Done on both beta and v1.0. Thanks!

**Comment by JulCsc** [comment]
> This pull request has conflicting changes, the author must resolve the conflicts before this pull request can be merged.

@angelgolfer-ms @Lauragra - please advise.

**Comment by angelgolfer-ms** [comment]
Merged to main after @FaithOmbongi helped fix the merge conflicts. Just so this PR won't have to run into another unexpected wave of merge conflicts to the TOC files.

**Pull request merged**